docs/source/coremltools.converters.mil.mil.ops.defs.rst

```diff
@@ -130,6 +130,7 @@ linear
 
    .. autoclass:: linear
    .. autoclass:: matmul
+   .. autoclass:: einsum
```

Comment:
Thanks for adding this. 
I was wondering if it could be set up in a way that we don't need to add an op here to be picked up by for documentation, it would be great if it was automatic.